apps/shared/avifutil.h

```diff
@@ -24,6 +24,7 @@ void avifImageDump(avifImage * avif, uint32_t gridCols, uint32_t gridRows);
 void avifContainerDump(avifDecoder * decoder);
 void avifPrintVersions(void);
 void avifDumpDiagnostics(const avifDiagnostics * diag);
+int avifQueryMaxThreads(void);
```

Comment:
This function should be named `avifQueryNumProcessors()` or `avifQueryNumCpus()` because that's what it returns ("to use all available cores").

We may want to document that this returns 1 if we cannot get the number of processors.

Reply:
I've updated both of these.